stdlib/public/Concurrency/TaskGroup.cpp

```diff
@@ -1150,7 +1158,7 @@ void AccumulatingTaskGroup::offer(AsyncTask *completedTask, AsyncContext *contex
   // This is wasteful, and the task completion function should be fixed to
   // transfer ownership of a retain into this function, in which case we
   // will need to release in the other path.
-  lock(); // TODO: remove fragment lock, and use status for synchronization
+  lock();
```

Comment:
By now it's not really helpful to have this TODO on every line where the lock is used.